docs/usage.rst

```diff
@@ -91,6 +91,18 @@ KafkaProducer
     for _ in range(100):
         producer.send('my-topic', b'msg')
 
+    def on_send_success(record_metadata):
+        print(record_metadata.topic)
+        print(record_metadata.partition)
+        print(record_metadata.offset)
+
+    def on_send_error(excp):
+        log.exception()
```

Comment:
Just doublechecking, since I'm not sure how this works:
Is the send exception always going to be the most recent one raised? If you screwed up your callback somehow and threw an exception, it would hide the send exception.

 Perhaps it'd be better to illustrate how to pass the `excp` into `log.error()`? This is honestly less of a `kafka-python` thing, and more just showing clearly how python logging works, as we often get questions on the issue tracker about that as well... 

Just a thought, I do not have a strong opinion, and I am not even 100% sure how this works in this particular example as I haven't played with it...

Reply:
I agree -- `log.exception()` may not always work here because we don't guarantee that the errback is evaluated within the context of a exception handler. It would be better to pass the exception explicitly:
```
def on_send_error(excp):
    log.error('I am an errback', exc_info=excp)
```